src/components/OptionRow.js

```diff
@@ -205,6 +206,16 @@ const OptionRow = (props) => {
                                     </Text>
                                 </View>
                             ) : null}
+                            {props.option.brickRoadIndicator === CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR && (
+                                <View style={[[styles.alignItemsCenter, styles.justifyContentCenter]]}>
```

Comment:
Is there a reason for the double brackets `[[ ]]` on this style?

Reply:
Oh whoa weird I copied these styles from somewhere else, but the double brackets are definitely not necessary (and surprised it even worked).